app/cafe.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+from typing import Dict
+
+from app.errors import (
+    OutdatedVaccineError,
+    NotWearingMaskError,
+    NotVaccinatedError
+)
+
+
+class Cafe:
+    """General class representing cafe."""
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: Dict) -> str:
+        """Check if visitor can visit a cafe."""
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated!")
+
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date < datetime.now().date():
```

Comment:
The comparison `expiration_date < datetime.now().date()` assumes that `expiration_date` is a `date` object. Ensure that `expiration_date` is indeed a `date` object and not a string. If it's a string, you need to convert it to a `date` object using `datetime.strptime` or a similar method before comparison.